pyproject.toml

```diff
@@ -19,6 +19,8 @@ dataclasses = { version = "^0.7", python = ">=3.6, <3.7" }
 grpclib = "^0.3.1"
 jinja2 = { version = "^2.11.2", optional = true }
 protobuf = { version = "^3.12.2", optional = true }
+toml = { version = "^0.10.1", optional = true }
+sphinx = { version = "3.1.2", optional = true }
```

Comment:
I don't think these belong here. Documentation isn't a concern of the distributed library.

Reply:
I thought all dependencies had to go there whoops.